ui/src/app/components/status-alert-banner-maybe.tsx

```diff
@@ -35,7 +35,10 @@ export const StatusAlertBannerMaybe = () => {
   useEffect(() => {
     const getAlert = async () => {
       const statusAlert = await statusAlertApi().getStatusAlert();
-      if (!!statusAlert) {
+      if (
```

Comment:
The hiding/showing logic is slightly different for BEFORE_LOGIN and AFTER_LOGIN, because AFTER_LOGIN alerts an be dismissed.

Reply:
optional: could be `statusAlert?.alertLocation ===` instead